### ISSUE TITLE
VoiceOver navigation shows a blank line after each message

### ISSUE THREAD

**Issue by OzancanKaratas** [deltachat/deltachat-ios#1590]
Is this an interface bug?

I navigate with VoiceOver and read messages. VoiceOver doesn't speak after jumping to the next or previous message. I have to swipe twice to jump to the previous or next one.

**Comment by cyBerta** [comment]
It is probably the avatar icon in group conversations. Can you double check if you experience the same as well in one on one conversations?

The "empty lines" should be fixed, thanks for the hint!

**Comment by OzancanKaratas** [comment]
> Can you double check if you experience the same as well in one on one conversations?

Yes.
